src/components/NeFileInput.vue

```diff
@@ -7,15 +7,15 @@
 import { computed } from 'vue'
 import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
 import { faFileArrowUp } from '@fortawesome/free-solid-svg-icons'
-import NeProgressBar from '@/components/NeProgressBar.vue'
+import NeProgressBar from './NeProgressBar.vue'
 
 interface FileInputProps {
   modelValue?: File | File[] | null
   label?: string
-  invalidMessage: string
-  dropzoneLabel: string
-  progress: number
-  showProgress: boolean
+  invalidMessage?: string
+  dropzoneLabel?: string
```

Comment:
This label is always visible, I'd remove the default value and set it as required in order to internationalize it correctly

Reply:
No problem.

Reply:
You mean inside the project?

Reply:
Sure